### ISSUE TITLE
OOM loss/mse

### ISSUE THREAD

**Issue by mqtjean** [sophie091524/Noise-Reduction-in-ECG-Signals#2]
Hi sophie091524, hi everyone, 

I was trying to use your DAE algorithm. During the training phase I got an Out Of Memory error despite setting batch_num = 1.

The error seems to come from the loss function (precisely the loss.mse function), trying to allocate 21 Gb of data.

I created my own training set with thirty 1024 sampled ECG paired with their noisy versions.

Is there a reasonable explanation to this enormous need of memory ?

**Comment by 1457209720** [comment]
您好，请问您的问题解决了吗

**Comment by mqtjean** [comment]
Hi thank you for your answer, 
There were probably a memory leak - I couldn't solve the problem but I found another way to denoise my ECG signal that was sufficient enough for what I needed.

**Comment by 1457209720** [comment]
您好，您是如何建立数据集的，论文里相关描述并不清楚，您可以解答一下吗？

**Comment by mqtjean** [comment]
Hi, I didn't train this neural network
